utils/datasets.py

```diff
@@ -67,14 +94,20 @@ def __len__(self):
 
     @abc.abstractmethod
     def __getitem__(self, idx):
-        """Get the image of `idx`
+        """Get the image of `idx`.
+
         Return
         ------
         sample : torch.Tensor
             Tensor in [0.,1.] of shape `img_size`.
         """
         pass
 
+    @abc.abstractmethod
+    def downlaod(self):
+        """Dowanload the dataset. """
+        pass
+
```

Comment:
Is this function just a placeholder at the moment?

Reply:
`@abc.abstractmethod`means that all objects that inherit from this class have to define that method (to be more precise: it's not possible to instantiate an object with an abstract method)

So because no other classes have a method called (with mispelling) `downlaod`, it should break the code. But I forgot that when you define an abstract method you shouldn't use `pass`. Because `pass` is a special keyword that defined a function without defining it. So actually the code doesn't break because they all have a method called `downlaod` which does nothing. 

TL;DR: thanks I need to remove `pass` and change the typo

Reply:
Let me know when you've done changes and I'll merge

Reply:
actually, the `pass` was correct (sorry I commented before testing haha) it was `abc.ABC` missing. SOlved and merge